oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -71,7 +77,10 @@ public class GoogleCredentials extends OAuth2Credentials implements QuotaProject
    * @return the credentials instance
    */
   public static GoogleCredentials create(AccessToken accessToken) {
-    return GoogleCredentials.newBuilder().setAccessToken(accessToken).build();
+    return GoogleCredentials.newBuilder()
```

Comment:
deprecate this constructor in favor of the builder?

Reply:
Done